src/components/Certificate.tsx

```diff
@@ -95,11 +95,7 @@ export const CertificateComponent = ({
         </Button>
         <div className="flex items-center justify-center">
           <Button
-<<<<<<< tailwind-prettier
```

Comment:
where did this get in?
@siinghd

Reply:
It was a merge conflit so I have allowed a incoming one

Reply:
@hkirat tailwind prettier was  here https://github.com/code100x/cms/pull/770/